back-end/API/app/services/jobOffer_service.py

```
@@ -1,176 +1,152 @@
+        if not original_job_offer.isApproved and (original_job_offer.title != updated_job_offer.title
+                                        or original_job_offer.description != updated_job_offer.description 
+                                        or original_job_offer.offerDebut != updated_job_offer.offerDebut 
+                                        or original_job_offer.address != updated_job_offer.address 
+                                        or original_job_offer.dateEntryOffice != updated_job_offer.dateEntryOffice 
+                                        or original_job_offer.deadlineApply != updated_job_offer.deadlineApply
+                                        or original_job_offer.email != updated_job_offer.email
+                                        or original_job_offer.hoursPerWeek != updated_job_offer.hoursPerWeek
+                                        or original_job_offer.offerLink != updated_job_offer.offerLink
+                                        or original_job_offer.salary != updated_job_offer.salary
+                                        ):
+            return True
```

Ruff (0.15.13)

[warning] Missing explicit `return` at the end of function able to return non-`None` value

Add explicit `return` statement

(RET503)